include/event_manager/event_manager.h

```diff
@@ -0,0 +1,317 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+/** @file
+ * @brief Event manager header.
+ */
+
+#ifndef ZEPHYR_INCLUDE_EVENT_MANAGER_EVENT_MANAGER_H_
+#define ZEPHYR_INCLUDE_EVENT_MANAGER_EVENT_MANAGER_H_
+
+
+/**
+ * @defgroup event_manager Event Manager
+ * @brief Event Manager
+ *
+ * @{
+ */
+
+#include <zephyr.h>
+#include <zephyr/types.h>
+#include <sys/reboot.h>
+#include <sys/__assert.h>
+#include <logging/log_ctrl.h>
+
+#include <event_manager/event_manager_priv.h>
```

Comment:
i started to wonder if event_manager is not too generic name, especially that there are structures like `event_header`. Maybe it should be additionally prefixed with `app_`. `app_event` tells more than just `event`.

Reply:
Maybe lets first introduce event manager to Zephyr? We could thane change name.